website/yarn.lock

```diff
@@ -9,35 +9,25 @@
   dependencies:
     "@babel/highlight" "^7.0.0"
 
-"@babel/code-frame@^7.10.4":
```

Comment:
It's something wrong with the Yarn lockfile, because dependencies are downgraded. Is your branch up to date with the `master`?